app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+
+require_relative styles.css
```

Comment:
This is not needed - your styles.css will be required and compiled automatically by the line `= require_tree .`.